org-formation/090-systems-manager/Scheduled-Script-Automation.yaml

```diff
@@ -50,8 +50,8 @@ Parameters:
   ScriptParameters:
     Type: String
     Description: >-
-      space delimited list of environment name-value pairs in the format name:value,
-      e.g., ENVNAME1:envval1 ENVNAME2:envval2
+      space delimited list of environment name-ssm parameter store pairs in the format env-name:ssm-name,
```

Comment:
hmm, every parameter needs to have a corresponding entry in the SSM?  What if you want to define parameters that are not secrets?

Reply:
Today the non-secret param's would be stored as secrets.  If that becomes limiting we can modify this automation.

Reply:
Not a fan, but i guess we can leave until later.